src/components/LogRow/AnsiiRow/index.tsx

```diff
@@ -11,13 +16,33 @@ const AnsiiRow = forwardRef<any, BaseRowProps>((rowProps, ref) => {
   // TODO: EVG-17535
   // This should be replaced with a collapsible component
   const lineIndex = Array.isArray(line) ? line[0] : line;
+  const lineContent = getLine(lineIndex);
+  if (!lineContent) return null;
   return (
     <BaseRow wrap={wrap} {...listRowProps} ref={ref}>
-      <span data-cy="ansii-row">{getLine(lineIndex)}</span>
+      <ProcessedAnsiiRow lineContent={lineContent} />
     </BaseRow>
   );
 });
 
+interface ProcessedAnsiiRowProps {
+  lineContent: string;
+}
+const ProcessedAnsiiRow: React.FC<ProcessedAnsiiRowProps> = ({
+  lineContent,
+}) => {
+  const memoizedLogLine = useMemo(() => {
+    if (!lineContent) return null;
```

Comment:
methinks you can delete this since it seems like it should be guaranteed to not be undefined

```suggestion

```